src/components/GordonDialogBox/GordonDialogBox.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../vars';
+
+.gc360_gordondialogbox {
```

Comment:
You created this file, but didn't edit the GordonDialogBox component at all? Or did you not commit the changes to the component yet?

Reply:
This might have been created as part of Sharon's branch, I haven't looked at this in detail yet, so correct, this is unfinished

Reply:
We'll look at this in detail when we get to it... is it okay to leave in for now?

Reply:
If it's not used, I don't think we should add it.  It will require attention to delete later, and distract people looking for things from what they really want.  Save it somewhere (or just leave it in whatever branch it came from).  But don't merge it into develop.

Reply:
I think that it would be better to remove it from the pull request so that we aren't merging unused code that will need to be reviewed later.

Reply:
So, we'll probably be adding a CSS file to Gordon Dialog Box anyway...  so the attention to check it later will already be given.  Would it be sufficient to add a comment to the CSS file for now, since it will likely be used later @EjPlatzer @russtuck ?

Reply:
> So, we'll probably be adding a CSS file to Gordon Dialog Box anyway... so the attention to check it later will already be given. Would it be sufficient to add a comment to the CSS file for now, since it will likely be used later @EjPlatzer @russtuck ?

@EjPlatzer thoughts on this?

Reply:
> > So, we'll probably be adding a CSS file to Gordon Dialog Box anyway... so the attention to check it later will already be given. Would it be sufficient to add a comment to the CSS file for now, since it will likely be used later @EjPlatzer @russtuck ?
> 
> @EjPlatzer thoughts on this?

If you're going to add it later, than it can be included in that pull request. It creates a needlessly confusing history for the file to have it created in git before it's referenced anywhere in the project. I think it should be removed before this PR is merged. We should, in general, strive for clean pull requests that are self-contained and leave the code cleaner than it was before.

Reply:
Okay, I've deleted it...